Source/Target.swift

```diff
@@ -49,14 +49,23 @@ public enum StructTarget: TargetType {
 
 /// Represents an HTTP method.
 public enum Method: String {
-    case GET, POST, PUT, DELETE, OPTIONS, HEAD, PATCH, TRACE, CONNECT
+
+    case options = "OPTIONS"
```

Comment:
No reason, was using [Alamofire as a reference](https://github.com/Alamofire/Alamofire/blob/e8bca7e40a861327366c5ccba42a688d17ec482a/Source/ParameterEncoding.swift#L30-L40)

Should I reorder it back to how it was?